htdocs/compta/paiement/list.php

```diff
@@ -275,6 +278,8 @@
 }
 $param .= (GETPOST("orphelins") ? "&orphelins=1" : '');
 $param .= ($search_ref ? "&search_ref=".urlencode($search_ref) : '');
+$param .= ($search_date_start ? "&search_date_start=".urlencode($search_date_start) : '');
```

Comment:
We must always keep the detail of the 3 component (day, month, year) when forgin url params for search component so we should have
if ($search_date_startday) {

if ($search_date_startmonth) {
..

Reply:
Already fix